src/django/pfb_analysis/management/commands/update_status.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand
+
+from pfb_analysis.models import AnalysisJobStatusUpdate, AnalysisJob
+
+
+class Command(BaseCommand):
+    help = "Update status during an analysis job"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('job_id')
+        parser.add_argument('status')
+        parser.add_argument('step', nargs='?')
```

Comment:
Still on the fence about `message` continuing to exist at all, but requiring `step` seems sensible.  How about
```
        parser.add_argument('job_id')
        parser.add_argument('status')
        parser.add_argument('step')
        parser.add_argument('message', nargs='?', default='')
```
?